src/index.html

```diff
@@ -21,7 +21,8 @@
     passageEndpoint: 'http://localhost:5001',
     environment: 'development',
     ingressBaseDomain: 'k8s.sample.io',
-    awsCapabilitiesJSON: '{"m3.large":{"description":"M3 General Purpose Large","memory_size_gb":7.5,"cpu_cores":2,"storage_size_gb":32},"m3.xlarge":{"description":"M3 General Purpose Extra Large","memory_size_gb":15,"cpu_cores":4,"storage_size_gb":80},"m3.2xlarge":{"description":"M3 General Purpose Double Extra Large","memory_size_gb":30,"cpu_cores":8,"storage_size_gb":160}}',
+    awsCapabilitiesJSON:
```

Comment:
This would need to stay on a single line. We're doing some find/replace in the index.html to set production values for this during a startup script: https://github.com/giantswarm/happa/blob/master/scripts/start.sh

Reply:
@oponder I have discussed this with Oriol last week. The question was: Why do we need to inject this at build time? We can maintain a list of all known instance types in happa. The info endpoint delivers the list of allowed types (only names) and the default type. That would simplify some things.

I don't see this reflected in the PR yet.

Reply:
We want to have a single source of truth for it (the installations repo), thats why in dev it is actually 'less' than what we have in prod. To really indicate that this is not the real list.

When we add a new instance type to the installations repo, I would like to avoid having to make changes in any other repos.

Reply:
Oh and the reason this info is 'extracted' to the installations repo is because `cluster-service` also makes use of it.

Reply:
We can fetch it in build time, can't  we?